packages/sync-actions/src/utils/copy-empty-array-props.js

```diff
@@ -15,13 +15,49 @@ export default function copyEmptyArrayProps(oldObj = {}, newObj = {}) {
           ...newObj,
           ...nextObject,
         }
+
+        if (key === 'custom') {
+          return {
+            ...merged,
+          }
+        }
+
+        if (Array.isArray(value) && newObj[key] && newObj[key].length >= 1) {
+          /* eslint-disable no-plusplus */
+          for (let i = 0; i < newObj[key].length; i++) {
+            if (
+              !isNil(newObj[key][i]) &&
+              typeof newObj[key][i] === 'object' &&
+              !isNil(newObj[key][i].id)
+            ) {
+              /* eslint-disable eqeqeq */
+              const foundObject = value.find((v) => v.id == newObj[key][i].id)
```

Comment:
Get same `object` inside the `oldObj` array

Reply:
what if we `Number(foo) === Number(bar)` and get rid of the eslint-disable. would that help?
I recommend you test it.

Reply:
Hmm, yep, that definitely will work, Just another dynamic types 😄